packages/quantic/force-app/main/default/lwc/quanticRecentQueries/quanticRecentQueries.js

```diff
@@ -0,0 +1,68 @@
+import { LightningElement, api, track } from 'lwc';
+import { registerComponentForInit, initializeWithHeadless } from 'c/quanticHeadlessLoader';
+import { getItemfromLocalStorage, setIteminLocalStorage } from 'c/quanticUtils';
+
+export default class QuanticRecentQueries extends LightningElement {
+  /** @type {string} */
+  localStorageKey = 'quantic-recent-queries';
+  /** @type {import("coveo").RecentQueriesState} */
+  @track state = {
+    queries: getItemfromLocalStorage(this.localStorageKey) ?? [],
+    maxLength: 10,
```

Comment:
Actually the maxLength needs to be configurable via @api prop. I'll post my new recommendation :P

Reply:
so we keep the default value as 10 and the user could set the maxLength to the value as needed?

Reply:
Yeah like 
```
@api maxLength = 10;
```

and in the initialize function

```
options: {
  maxLength: this.maxLength
}
```

What do you think?

Reply:
👌